### PR TITLE
Adds service object for updating stock prices and uses the correct method for getting the stock price

### PR DESCRIPTION
**Pull request by brandomoreno06**
### Story
Price of each is required for a user to complete a trasaction. Initially, I have used client.price(<Stock Symbol>) to get the prices using the sandbox end point for testing. After testing client.price(<Stock Symbol>) using the production endpoint https://cloud.iexapis.com/v1 locally, I received a error message that the data is not available for free tier accounts. So instead, I tried to use another method, client.quote(<Stock Symbol>),  which returned an object that includes an attribute for the price.

For the seed.rb file, I extracted the code for updating the price right after fetching the stocks symbols, and created a service object for updating the stock prices. This will ensure the dryness of code as this service object will be used not only during seeding of initial prices, but also when updating the prices on a scheduled basis.

### Screenshot
image

### References
https://github.com/dblock/iex-ruby-client
https://docs.avionschool.com/avn-mcourse/-MYjabeTubdULkJ4UAcG/back-end-course-further-reading/ror-service-objects

### CODE REVIEW

**Review by bryanarboledaa — APPROVED** [review]
LGTM, approve.

File: app/services/update_stock_price.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+require 'iex-ruby-client'
```
Comment (analyn-cajocson) [inline app/services/update_stock_price.rb]: No need to add this `require 'iex-ruby-client'`

**Review by analyn-cajocson — CHANGES_REQUESTED** [review]

File: app/services/update_stock_price.rb (hunk 1 above)
Comment (brandomoreno06) [reply app/services/update_stock_price.rb]: It seems strange, as when I don't add `require 'iex-ruby-client'`, the code fails as it is looking for the module. So I added that to explicitly import it. Might there be a problem with my setup? Please enlighten me. TYIA.

File: app/services/update_stock_price.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+class UpdateStockPrice
+  def self.call(...)
+    new(...).execute
+  end
+
+  def execute
```
Comment (analyn-cajocson) [inline app/services/update_stock_price.rb]:
```suggestion
class UpdateStockPrice

  def self.execute
```

**Review by analyn-cajocson — CHANGES_REQUESTED** [review]

**Pull request merged**